python/cuml/test/test_random_forest.py

```diff
@@ -244,22 +224,37 @@ def test_rf_regression_default(datatype, column_info, nrows):
 
 
 @pytest.mark.parametrize('datatype', [np.float32])
-@pytest.mark.parametrize('column_info', [unit_param([20, 10]),
-                         quality_param([200, 100]),
-                         stress_param([500, 350])])
-@pytest.mark.parametrize('nrows', [unit_param(500), quality_param(5000),
-                         stress_param(500000)])
-def test_rf_classification_seed(datatype, column_info, nrows):
+def test_rf_classification_seed(dataset, datatype):
 
-    ncols, n_info = column_info
-    X, y = make_classification(n_samples=nrows, n_features=ncols,
-                               n_clusters_per_class=1, n_informative=n_info,
-                               random_state=0, n_classes=2)
+    X, y = dataset
     X = X.astype(datatype)
     y = y.astype(np.int32)
     X_train, X_test, y_train, y_test = train_test_split(X, y, train_size=0.8,
                                                         random_state=0)
-    for i in range(20):
+
+    cuml_model = curfc()
```

Comment:
it feels weird to just add the default classification test here like this. 
I believe we can remove the part of default model test as the default model is used in the `test_rf_classification_float64`  tests. 
Furthermore, we should be good to remove the `test_rf_regression_default`  test as well, since the `test_rf_regression_float64` test uses the default model, here as well.